.github/workflows/benchmark-flow.yml

```diff
@@ -0,0 +1,117 @@
+name: Run a Benchmark Flow
+
+on:
+  workflow_call:
+    inputs:
+      container:
+        type: string
+        default: redisfab/rmbuilder:6.2.7-x64-bullseye
+      module_path:
+        type: string
+        default: bin/linux-x64-release/src/redisgraph.so
+      profile_env:
```

Comment:
not sure what `profile_env` stands for

Reply:
removed it. We don't profile on graph on CI